cmd/ef_tests/levm/utils.rs

```diff
@@ -17,3 +17,16 @@ pub fn load_initial_state(test: &EFTest) -> (EvmState, H256) {
         genesis.get_block().header.compute_block_hash(),
     )
 }
+
+// If gas price is not provided, calculate it with current base fee and priority fee
+pub fn effective_gas_price(test: &EFTest, tx: &&EFTestTransaction) -> U256 {
+    match tx.gas_price {
+        None => {
+            let current_base_fee = test.env.current_base_fee.unwrap();
+            let priority_fee = tx.max_priority_fee_per_gas.unwrap();
+            let max_fee_per_gas = tx.max_fee_per_gas.unwrap();
+            std::cmp::min(max_fee_per_gas, current_base_fee + priority_fee)
+        }
```

Comment:
Let's enforce the `max_priority_fee_per_gas`, `max_fee_per_gas`, and `current_base_fee` to be required if `gas_price` is not present in the transaction test data. Do not unwrap here but return an `EFTestRunnerError`.

Reply:
Done in https://github.com/lambdaclass/ethrex/pull/1345/commits/f5dd02ef91a6517c5176d56d514ccbac30ad1634
Let me know if you think it should be other type of error.